mappings/net/minecraft/component/type/ItemEnchantmentsComponent.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/unmapped/C_qlclfxvn net/minecraft/component/type/ItemEnchantmentsComponent
+	FIELD f_nekwvric FULL_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_osezxuts DEFAULT Lnet/minecraft/unmapped/C_qlclfxvn;
+	FIELD f_svrotkyo LEVEL_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_weawpons LEVELS_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_zenuxhpu shownInTooltip Z
```

Comment:
```suggestion
	FIELD f_zenuxhpu showInTooltip Z
```

Reply:
for fields my name follows convention better, that would be the method name